presentation/error_404_view.py

```diff
@@ -0,0 +1,9 @@
+import dash_html_components as html
+
+
+class Error404View:
+    @staticmethod
+    def make_layout() -> html.Div:
```

Comment:
Why not have a single `ErrorView` class and pass the error code as an argument to `make_layout`?

```
class ErrorView:
    ERROR_MESSAGE = {
         404: "404 Page not found",
         500: "500 Internal server error"
    }

    @classmethod
    def make_layout(cls, code: int) -> html.Div:
        return html.Div(
            children=[html.H1(children=cls.ERROR_MESSAGE[code],
                                           style={"textAlign": "center", "marginBottom": 50})],
        )
```